src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,39 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01 13:52:04 UTC
+
+| Fork name | URL | Available platforms |
+|---|---|---|
+| Wizard's Den | [Link](https://wizards.cdn.spacestation14.com/fork/wizards) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Harmony 14 | [Link](http://cdn.harmony14.com/fork/harmony/) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Solaris 14 | [Link](https://ss14-cdn.mw-gc.com/fork/MWGSolaris14) | Windows, macOS (x64/Intel), Linux (x64/ARM64) |
+| Axolotl MRP | [Link](https://axolotl.yuniiworks.de/cdn/fork/axolotl/) | Linux (x64) |
+
+FAQ:
+
+- The fork I want is not listed here.
+	- You will need to contact the fork you want a server build for and ask them if they provide one. If they do suggest them to add themselves here.
+	
+- The fork I want is listed here, but my platform is not available for download.
+	- You will need to contact the fork asking if they are willing to build for your platform.
+	
+- The fork does not provide prebuilds (or any other issue where you are unable to retrive a server prebuild)
+	- Your only choice is to compile the forks codebase yourself if available. Instructions can be found [here](../general-development/setup/server-hosting-tutorial.md#level-2-server-with-custom-code)
+
+- I'm a server host and I want to add myself here.
+	- You are free to make a PR yourself in the docs or ping @vasilisiscool (322708417540259841) in #hosting on discord with all the information needed.
+
+<!-- 
+Rules for listing:
+1. Must have a decent playerbase on the hub and been on for at least a week.
```

Comment:
Should probably clearly define "decent playerbase" since people may have different opinions on what's considered a decent playerbase.

Reply:
Ours (Solaris 14) probably doesn't even meet that out of the gate. We have roughly five regulars and are still working on our own content to distinguish our fork from Wizden. I wouldn't pitch a fit if we were removed, but think it is awesome that we are on the list :)